L1TriggerConfig/L1TConfigProducers/scripts/runL1-O2O-scramTests.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/sh
+#set -x
+
+xflag=0
+CMS_OPTIONS=""
+KEY_CONTENT=""
+TAG_UPDATE=""
+UNSAFE=""
+
+while getopts 'xfk:t:u:d:h' OPTION
+  do
+  case $OPTION in
+      x) xflag=1
+          ;;
+      f) CMS_OPTIONS=$CMS_OPTIONS" forceUpdate=1"
+          ;;
+      k) KEY_CONTENT=$KEY_CONTENT" subsystemLabels=$OPTARG"
+          ;;
+      t) if [ -z $TAG_UPDATE ] ; then TAG_UPDATE="tagUpdate="; else TAG_UPDATE=$TAG_UPDATE","; fi
+         TAG_UPDATE=$TAG_UPDATE$OPTARG
+          ;;
+      u) if [ -z $UNSAFE ] ; then UNSAFE="unsafe="; else UNSAFE=$UNSAFE","; fi
+         UNSAFE=$UNSAFE$OPTARG
+	 ;;
+      d) if [ -z $DROPSYSTEMS ] ; then DROPSYSTEMS="dropFromJob="; else DROPSYSTEMS=$DROPSYSTEMS","; fi
+         DROPSYSTEMS=$DROPSYSTEMS$OPTARG
+         ;;
+      h) echo "Usage: [-xf] runnum tsckey"
+          echo "  -x: write to ORCON instead of sqlite file"
+          echo "  -f: force IOV update"
+          echo "  -k: limit update to the specific systems (default are all, which is equivalent to -k uGT,uGTrs,GMT,EMTF,OMTF,BMTF,CALO)"
+          echo "  -t: override tag name as TYPE:NEW_TAG_BASE (e.g. -t L1TCaloParams:Stage2v1)"
+          echo "  -u: lift transaction safety: carry on even problems are encountered (e.g. -u EMTF,OMTF,CALO)"
+          echo "  -d: dron these systems form the job: Dont create WriterProxyT for these systems in the PayloadWriter Constructor (e.g. -d EMTF,OMTF,CALO)"
+          exit
+          ;;
+  esac
+done
+shift $(($OPTIND - 1))
+
+runnum=$1
+tsckey=$2
+rskey=$3
+
+
+echo CMSSW_BASE = $CMSSW_BASE
+echo PWD = $PWD
+ls -a
+
+export TNS_ADMIN=/etc
+echo "INFO: ADDITIONAL CMS OPTIONS:  " $CMS_OPTIONS $KEY_CONTENT $TAG_UPDATE
+
+ONLINEDB_OPTIONS="onlineDBConnect=oracle://cms_omds_adg/CMS_TRG_R onlineDBAuth=./"
+PROTODB_OPTIONS="protoDBConnect=oracle://cms_orcon_adg/CMS_CONDITIONS protoDBAuth=./"
+
+[ "$KEY_CONTENT" == "subsystemLabels=uGT,uGTrs" ] \
+    && DATA_DIR=$CMSSW_BASE/l1o2o_L1TMenu_data \
+    || DATA_DIR=$CMSSW_BASE/l1o2o_L1TSubs_data
+mkdir $DATA_DIR
+cp $CMSSW_BASE/src/L1TriggerConfig/L1TConfigProducers/data/l1config.db $DATA_DIR/
```

Comment:
as `l1config.db` is part of cms-data , so cleanup this PR and remove it from the history. Also use $CMSSW_SEARCH_PATH to find the data file e.g. 
```
for dir in $(echo $CMSSW_SEARCH_PATH | tr ':' '\n') ; do
  [ -e ${dir}/L1TriggerConfig/L1TConfigProducers/data/l1config.db ] || continue
  cp ${dir}/L1TriggerConfig/L1TConfigProducers/data/l1config.db $DATA_DIR/
  break
done
```